src/LineBot/LineBot.cs

```diff
@@ -489,6 +489,37 @@ public async Task<ILineBot> Reply(string replyToken, params ISendMessage[] messa
             return this;
         }
 
+        /// <summary>
+        /// Sets the default rich menu, which is displayed to all users who have added your bot as a friend and are not linked to any per-user rich menu.
+        /// </summary>
+        /// <param name="richMenu">The rich menu response.</param>
+        /// <returns>.</returns>
+        public async Task<ILineBot> SetDefaultMenu(IRichMenuResponse richMenu)
+        {
+            Guard.NotNull(nameof(richMenu), richMenu);
+
+            await SetDefaultRichMenu(richMenu.Id);
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the default rich menu, which is displayed to all users who have added your bot as a friend and are not linked to any per-user rich menu.
+        /// </summary>
+        /// <param name="richMenuId">The rich menu id.</param>
+        /// <returns>.</returns>
+        public async Task<ILineBot> SetDefaultRichMenu(string richMenuId)
+        {
+            Guard.NotNullOrEmpty(nameof(richMenuId), richMenuId);
+
+            var emptyObject = new object();
+            var content = CreateStringContent(emptyObject);
```

Comment:
Wondering if we could not just set content to `{}` instead?

Reply:
Current implementation creates content as {}. 
The CreateStringContent("{}"); will create content as "{}". 

I can simplify 2 lines into 1 like this: var content = CreateStringContent(new object());
What do you prefer?

Reply:
Just took another look at the API and it seems that you should not post an object? Maybe pass `null` to `PostAsync` instead?